docs/source/data.rst

```diff
@@ -25,8 +25,13 @@ Two types of data must be distinguished:
       ``metadata.yml``. It has a ``data_url`` key that references to the remote/online repository where
       station's raw data are stored. At this URL, a single zip file provides all data available for a given station.
 
-How to download the DISDRODB metadata and issues archive ?
---------------------------------------
+
+Data transfer upload and download schema :
+
+.. image:: /static/transfer.png
```

Comment:
I have some suggestion to improve the figure (which is already great). 
- Instead of `DisdroDB official repository` --> `DISDRODB metadata archive repository`
- Add background to git command. For example `git clone + pull`
- Maybe the term `update` is more appropriate than `upload` for the metadata archive.
- Maybe you can add also a dashed arrow from `DISDRODB Download/Upload Utility` to `Stations raw data` and `Personal online repository`